lib/emails/invitation.ts

```diff
@@ -26,28 +28,30 @@ const sendEmail = (invitation: Invitation, provider: EmailProvider): Promise<voi
   new Promise((resolve, reject) => {
     const { transport, from } = provider;
 
+    const { language: t } = invitation;
     const invitationHtml = html(invitation);
     nodemailer.createTransport(transport).sendMail(
       {
         from: `Cal.com <${from}>`,
         to: invitation.toEmail,
-        subject:
-          (invitation.from ? invitation.from + " invited you" : "You have been invited") +
-          ` to join ${invitation.teamName}`,
+        subject: invitation.from
+          ? t("user_invited_you", { user: invitation.from, teamName: invitation.teamName })
+          : t("you_have_been_invited", { teamName: invitation.teamName }),
         html: invitationHtml,
         text: text(invitationHtml),
       },
       (error) => {
         if (error) {
           console.error("SEND_INVITATION_NOTIFICATION_ERROR", invitation.toEmail, error);
-          return reject(new Error(error));
+          return reject(new Error(error.message));
```

Comment:
I'm actually relatively certain this previously handled the new Error(<string>error); as throwables can be an Error or string object - this is quite likely to fail (needs to handle the different types, string or Error) - also; this is also showing as a typescript error on my machine.

Reply:
Passing an `Error` object to another new Error causes this: `Error: Error: <message>` - but calling .message on a string fails a bit harder, too.